src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -144,7 +156,16 @@ export const useEstimateTransactionGas = ({
 
       if (isOffChainSignature && !isCreation) {
         setGasEstimation(
-          getDefaultGasEstimation(EstimationStatus.SUCCESS, '1', '1', isExecution, isCreation, isOffChainSignature),
+          getDefaultGasEstimation(
```

Comment:
Would it not make more sense to instead of having these as separate arguments, send them in an object? It is a bit confusing to follow.